src/typeInference/annotateTypeSwitchOperator.ts

```diff
@@ -0,0 +1,7 @@
+import { SequenceType } from '../expressions/dataTypes/Value';
+import { IAST } from '../parsing/astHelper';
+
+// TODO: Annotation not yet implemented. The docs don't mention it so we don't know the return types.
+export function annotateTypeSwitchOperator(ast: IAST): SequenceType {
+	return undefined;
```

Comment:
Feel free to return `item()*` if you are not sure. That way we have at least some typing and we will have fewer duplicate branches (`if (type === 'item()*' || type === undefined){...}`)